third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -130,8 +130,8 @@ function juLog() {
   # calculate vars
   asserts=$(($asserts+1))
   errors=$(($errors+$err))
-  time=`echo "${end} - ${ini}" | bc -l`
-  total=`echo "${total} + ${time}" | bc -l`
+  time=`echo "${end} ${ini}" | awk '{print $1 - $2}'`
```

Comment:
couldn't we also just use bash arithmetic? 
```bash
echo $((${end} - ${ini}))
echo $((${total} + ${time}))
```

I checked in the busybox image and this seems to work and is more readable.

Reply:
No, since the times are floating points. Bash doesn't have floating point arithmetic.